clients/lsp-nix.el

```diff
@@ -54,9 +54,38 @@
   :type 'string
   :package-version '(lsp-mode . "8.0.1"))
 
+(defcustom lsp-nix-nil-formatter nil
```

Comment:
can you use defcustom-lsp?

Reply:
Thanks, didn't know about it and it makes things much simplified.